opentracing-jaxrs2-itest/common/src/main/java/io/opentracing/contrib/jaxrs2/itest/common/AbstractServerTest.java

```diff
@@ -277,6 +282,40 @@ public void testSerializationResponseAndRequestWithBody() {
         assertResponseSerialization(parentSpan, serializationResponseSpan);
     }
 
+    @Test
+    public void testMultipleServerRequests() throws ExecutionException, InterruptedException {
+        int numberOfThreads = 10;
+        ExecutorService executorService = Executors.newFixedThreadPool(numberOfThreads);
+        List<Future<?>> futures = new ArrayList<>(numberOfThreads);
+        for (int i = 0; i < numberOfThreads; i++) {
+            futures.add(executorService.submit(new Runnable() {
+                @Override
+                public void run() {
+                    for (int j = 0; j < 10; j++) {
+                        ClientBuilder.newClient()
+                            .target(url("/hello/" + j))
+                            .request()
+                            .get();
+                    }
+                }
+            }));
+        }
+
+        for (Future<?> future: futures) {
```

Comment:
Could just use Awaitlity for the number of mockspans, as don't need the results from the futures, or a 2 sec buffer.

Reply:
I would like to close all resources anyways. Then awaitility is not needed. 

I have changed order of shutdown and awaittermination. Shutdown should be first.